sdks/python/tox.ini

```diff
@@ -326,3 +326,16 @@ commands =
   # Run all PyTorch unit tests
   # Allow exit code 5 (no tests run) so that we can run this command safely on arbitrary subdirectories.
   /bin/sh -c 'pytest -o junit_suite_name={envname} --junitxml=pytest_{envname}.xml -n 6 -m uses_pytorch {posargs}; ret=$?; [ $ret = 5 ] && exit 0 || exit $ret'
+
+[testenv:py{37,38,39,310}-xgboost-{160,170}]
+deps =
+  -r build-requirements.txt
+  160: torch>=1.6.0,<1.7.0
+  170: torch>=1.7.0
```

Comment:
Instead of torch, this should be installing xgboost, right? I think we also need to be installing datatable here, right?

Reply:
Ah, we also need to handle import exceptions like this - https://github.com/apache/beam/blob/aee2c844ad8f583dc161219f4ea0988d5a8860e9/sdks/python/apache_beam/ml/inference/pytorch_inference_it_test.py#L36 - and skip if datatable/xgboost aren't installed like this - https://github.com/apache/beam/blob/aee2c844ad8f583dc161219f4ea0988d5a8860e9/sdks/python/apache_beam/ml/inference/pytorch_inference_it_test.py#L63

Also, please add uses_xgboost to https://github.com/apache/beam/blob/aee2c844ad8f583dc161219f4ea0988d5a8860e9/sdks/python/pytest.ini#L29